nncf/experimental/post_training/samplers.py

```diff
@@ -39,9 +41,6 @@ def __iter__(self) -> Iterator[NNCFData]:
                 self.batch_indices[i], self.batch_indices[i + 1])
             yield batch
 
-    def __len__(self) -> int:
```

Comment:
What reason of these changes?

Reply:
I added __len__ method to the ABC class Sampler